src/python/tests/core/platforms/android/adb_test.py

```diff
@@ -135,13 +135,6 @@ def test_wait_for_device(self):
     adb.wait_for_device()
     self.assertEqual(adb.get_device_state(), 'device')
 
-  def test_remount(self):
```

Comment:
This test wont work without disable verity. this will require reboot, etc, this probably will go in device_test.py